### DISCUSSION TITLE
Get the id of a zone

### DISCUSSION THREAD

**Discussion by eperiana** [allenporter/pyrainbird#65] [General]
Hello,
I have read your protocol for integrating Rainbird into a home automation system but I was wondering:
How do you get the id of a zone?
Is there a function or a way to retrieve this id?

Kind regards

**Comment by allenporter** [comment]
I'm not sure I follow the question exactly, but you can get the list of zones from `get_available_stations` then get information about each zone (whether on or off) from `get_zone_states`.
